### DISCUSSION TITLE
Deploying ML Project using Python & CSV File

### DISCUSSION THREAD

**Discussion by kundusarnav** [github/india#729] [Students]
I made ML project on Stock Prediction by Python (.py) and excel (.csv). At first I used static web platform but later I realized that it must contain html file. After that I tried Azure ML platform but could not able to understand about it. Please guide me how to deploy ML project using Python and CSV . Also, it will be more effective if there will be mentoring session about this.

**Comment by badfreakout** [comment]
> I made ML project on Stock Prediction by Python (.py) and excel (.csv). At first I used static web platform but later I realized that it must contain html file. After that I tried Azure ML platform but could not able to understand about it. Please guide me how to deploy ML project using Python and CSV . Also, it will be more effective if there will be mentoring session about this.

@kundusarnav  Have u completed ur project?
